tests/profiles/googlefonts_test.py

```diff
@@ -4249,6 +4249,33 @@ def test_check_colorfont_tables():
     assert_PASS(check(ttFont),
                 f'with a good font without SVG or COLR tables.')
 
+def test_check_empty_glyph_on_gid1_for_colrv0():
+    """Put an empty glyph on GID 1 right after the .notdef glyph for COLRv0 fonts."""
+    check = CheckTester(googlefonts_profile,
+                        "com.google.fonts/check/empty_glyph_on_gid1_for_colrv0")
+
+    ttFont = TTFont(TEST_FILE("color_fonts/noto-glyf_colr_1.ttf"))
+    space_glyphname = "space"
+    if "uni0020" in ttFont['cmap'].tables[0].cmap:
```

Comment:
cmap subtable are a dict keyed by the unicode codepoint (int), with values the glyph name (it's actually a glyph index in the binary table), not the other way around, so this `if` will never be True. Perhaps you meant `if "uni02020" in cmap.values()` but even that is weird because that simply tells you that there's a glyph somewhere in the cmap that has that name, it doesn't tell you which codepoint it is mapped to. If you just want to check if the font contains a glyph named uni0020 without bothering to see if it's mapped to 0x0020, you can just check if it's in the TTFont.getGlyphOrder()

Reply:
Actually I will reduce this to use `space` right away since it's a test and I already know that the `space` glyph exists. This is a remnant of the previous check logic

Reply:
ok. BTW, I think you're using a COLRv1 test font for a check that's supposed to only matter for COLRv0